### DISCUSSION TITLE
리액트 서버 api 리스폰스 마이너 수정사항

### DISCUSSION THREAD

**Discussion by woohm402** [wafflestudio/seminar-2021#548] [Announcements]
생각해보니 지금 api대로면 ***"추가한 학생을 바로 선택하는"*** 기능을 구현하기가 상당히 번거로워져서, 리스폰스에 추가된 학생의 id값을 추가했습니다.

기존
```json
{
  "success": true
}
```

변경
```json
{
  "success": true,
  "id": 1
}
```

물론 `{ id: 1 }`은 예시이고, 1 말고 어떤 정수가 들어가게 될 거예요! [api 문서](https://p04fpgjlo3.execute-api.ap-northeast-2.amazonaws.com/swagger/#/%ED%95%99%EC%83%9D%20%EA%B4%80%EB%A6%AC%20API/StudentController_createStudent)에도 업데이트해 두었습니다

**Comment by 0114kek** [comment]
서버에 학생 정보를 요청했을때 profile_img 정보가 api문서와 달리 오지 않는것같습니다. 확인부탁드립니다!

**Comment by woohm402** [comment]
> 서버에 학생 정보를 요청했을때 profile_img 정보가 api문서와 달리 오지 않는것같습니다. 확인부탁드립니다!

세상에 그러네요 확인했습니다ㅠㅠ 퇴근하고 바로 수정하겠습니다 감사합니다!

**Comment by woohm402** [comment]
@0114kek 수정되었습니다 👍
